packages/design-system/src/components/Drawer/Drawer.tsx

```diff
@@ -92,10 +94,12 @@ export const Drawer = (props: DrawerProps) => {
         >
           {props.children}
         </div>
-        <div className="ds-c-drawer__footer">
-          <h4 className="ds-c-drawer__footer-title">{props.footerTitle}</h4>
-          <div className="ds-c-drawer__footer-body">{props.footerBody}</div>
-        </div>
+        {(props.footerTitle || props.footerBody) && (
```

Comment:
mgov doesn't seem to use footers in its Drawer. The way this was originally coded, the footer content was all optional, but if it was omitted the markup for the footer (divs and stuff) would still exist in DOM. This removes unneeded markup.